### DISCUSSION TITLE
Can someone help me with this, what I am supposed to do now that this appeared when I was updating Vlad diffusion?

### DISCUSSION THREAD

**Discussion by superprohero** [vladmandic/sdnext#1387] [Q&A]
Should I just close the folder and move on, or is there something I need to do?

![okdfjk;kf;lkw;](https://github.com/vladmandic/automatic/assets/133063465/0a9ee10f-8d3e-4822-aade-f4425f4541cc)

**Comment by vladmandic** [comment]
you already have a thread going, its bad etiquette to spam multiple threads. i'll reply on the original thread.
